src/components/structures/FilePanel.js

```diff
@@ -50,6 +50,9 @@ class FilePanel extends React.Component {
         if (room?.roomId !== this.props?.roomId) return;
         if (toStartOfTimeline || !data || !data.liveEvent || ev.isRedacted()) return;
 
+        const client = MatrixClientPeg.get();
+        client.decryptEvent(ev);
```

Comment:
seems weird to call a method called `decryptEvent` on something which may be not encrypted in the first place, either needs wrapping in an if check of `shouldAttemptDecryption` or the method could be renamed something like `decryptEventIfNeeded`